packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -306,7 +304,11 @@ function transform(codeGen: CodeGen): t.Expression {
         return codeGen.genIterator(iterable, iterationFunction);
     }
 
-    function computeAttrValue(attr: Attribute | Property, element: BaseElement): t.Expression {
+    function computeAttrValue(
+        attr: Attribute | Property,
+        element: BaseElement,
+        addLegacySanitizationHook: boolean
```

Comment:
Can we remove the legacy sanitization hook entirely? We've always described `setHooks`, `sanititizeAttribute`, and `sanitizeHtml` as ["experimental" APIs](https://github.com/salesforce/lwc/blob/52e0a2b3a0463df42baa161cfd369aa069d96303/packages/%40lwc/engine-core/README.md#sethooks).

We could remove these entirely and do a major version bump. (LWC v3.0.0)

Reply:
This will result is dropping security coverage. Custom components which are on legacy locker will loose their current sanitization capability. ProdSec will probably object to that.

Reply:
Gotcha, thanks for the explanation!